integration_test/lib/vnspec/vm.rb

```diff
@@ -248,6 +256,10 @@ def ipv4_address
         end
       end
 
+      def change_ipv4_address(new_address, new_prefix = 24)
```

Comment:
It's code that will be used in the simple_seg test once I've refactored the dhcp/static address configuration for CI interfaces.